config/locales/ar.yml

```diff
@@ -2,19 +2,19 @@ ar:
   site:
     title: Local Welcome
     hero_text: تعقد مؤسسة "لوكال ويلكم" اجتماعات شهرية في اماكن عامة لمساعدة اللاجئين لتعلم اللغة الانكليزية و التطوع ضمن المجتمع والعودة الى العمل
-    signed_in_as: Signed in as <a href="%{link}">%{email}</a>
-    admin_dashboard: Admin dashboard
+    signed_in_as: تسجيل الدخول كـ <a href="%{link}">%{email}</a>
+    admin_dashboard: لوحة التحكم
     sign_out: تسجيل الخروج
     sign_in: تسجيل الدخول
     sign_up: اشترك
     learn_more: اعرف أكثر
-    view_in_english: View in English
-    view_in_arabic: View in Arabic
+    view_in_english: عرض باللغة الانكليزية
+    view_in_arabic: عرض باللغة العربية
     footer:
-      email: Questions? Email us on <a href="mailto:localwelcometeam@gmail.com">localwelcometeam@gmail.com</a>
-      twitter: Follow Local Welcome on <a href="https://twitter.com/localwelcome">Twitter</a>
-      facebook: Like Local Welcome on <a href="https://www.facebook.com/LocalWelcome/">Facebook</a>
-      established: Local Welcome was established in 2015. Learn more <a href="%{link}">about the project</a>.
+      email: للاستفسارات? أرسل لنا على <a href="mailto:localwelcometeam@gmail.com">localwelcometeam@gmail.com</a>
+      تويتر: تابعونا على <a href="https://twitter.com/localwelcome">Twitter</a>
+      فيس بوك: تابعونا على <a href="https://www.facebook.com/LocalWelcome/">Facebook</a>
+      established: تم تأسيس Local Welcome في 2015. لمعرفة المزيد <a href="%{link}">about the project</a>.
```

Comment:
The "about the project" part within the `<a>` tag needs to be translated here too: 

```
established: تم تأسيس Local Welcome في 2015. لمعرفة المزيد <a href="%{link}">about the project</a>.
                                                                             ↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑
```